perfmetrics/scripts/ml_tests/pytorch/run_model.sh

```diff
@@ -49,12 +49,12 @@ logging:
     compress: true
 metadata-cache:
   ttl-secs: 1728000
+  stat-cache-max-size-mb: 1511
```

Comment:
1511 = HigherMiBs(1320000*1200 bytes)

Reply:
please round it off to 1600, otherwise it may confuse other developers.

Reply:
done